api/openapi_server/controllers/auth_controller.py

```diff
@@ -123,6 +123,44 @@ def signup():  # noqa: E501
 
     return response
 
+def signUpCoordinator():  # noqa: E501
+    """Signup a new Coordinator
+    """
+    if connexion.request.is_json:
+        body = connexion.request.get_json()
+
+    secret_hash = get_secret_hash(body['email'])
+
+    # Signup user
+    with Session(db_engine) as session:
+        user = db.User(email=body['email'])
+        session.add(user)
```

Comment:
why do we store a coordinator and not a host's email in postgres?

Reply:
We save the host as well. Not sure why Github decided not to highlight it in the changes but you can see it if you view the whole file

https://github.com/hackforla/HomeUniteUs/blob/2d1d17d37764cc0ecc231e01b4f2e531252853a3/api/openapi_server/controllers/auth_controller.py